python/pylibcudf/pylibcudf/io/avro.pyx

```diff
@@ -9,6 +9,9 @@ from pylibcudf.libcudf.io.avro cimport (
     read_avro as cpp_read_avro,
 )
 from pylibcudf.libcudf.types cimport size_type
+from rmm.pylibrmm.stream cimport Stream
+from pylibcudf.utils cimport _get_stream
```

Comment:
I recommend keeping imports sorted. This won't happen automatically in `.pyx` files (ruff doesn't support it) but it's good practice.

Reply:
This happens in a few files, please look over the diff to see them all.